### PR TITLE
Switch to Headless Chrome for running /apps tests

### PR DESCRIPTION
**Pull request by joshlory**
It's so fast 😭

I followed this guide: https://developers.google.com/web/updates/2017/06/headless-karma-mocha-chai (thank you @islemaster for the link!)

### CODE REVIEW

File: apps/README.md [hunk 1]
```diff
@@ -106,9 +106,7 @@ This will work on any of the test commands.
 
 ##### Debugging Tests #####
 
-To debug tests, your best bet is to run them in Chrome. Keep in mind that there
-can be subtle differences between Chrome and PhantomJS, so after fixing your
-test in Chrome, make sure it still works in PhantomJS. To run the tests in
+To debug tests, your best bet is to run them in Chrome. To run the tests in
 Chrome, use the `BROWSER` environment variable in conjunction with `WATCH`:
```
Comment (Bjvanminnen) [inline apps/README.md:120]: Is one able to debug directly in chrome headless?

**Comment by joshlory** [comment]
Note that we can't turn on Scratch rendering in Headless Chrome until they enable WebGL for headless: https://bugs.chromium.org/p/chromium/issues/detail?id=617551

File: apps/README.md (hunk 1 above)
Comment (joshlory) [reply apps/README.md:120]: I think we still need to keep these instructions for `BROWSER=Chrome ...`

**Comment by islemaster** [comment]
- Any idea if there's been a change to the memory overhead required to run these tests now?

**Comment by islemaster** [comment]
Can we tear

**Comment by islemaster** [comment]
Can we remove any of our phantomjs dependencies as well?  At least `karma-phantomjs-launcher`.  We have a few places that Ruby code uses phantom but that should use the phantom dependency in our gemfile.  Looks like `generateLevelImages.js` may still use it?

**Comment by islemaster** [comment]
There's also still a mention of manually installing `phantomjs` in our SETUP.md that should probably go away.

**Comment by islemaster** [comment]
Curiously, integration tests run quite a few tests but seem to abort in the middle of them:
```
03 08 2017 13:36:07.662:WARN [Chrome 60.0.3112 (Linux 0.0.0)]: Disconnected (1 times)

Finished in 22.73 secs / NaN secs

Warning: Task "karma:integration" failed. Use --force to continue.

Aborted due to warnings.
```

**Comment by joshlory** [comment]
I believe we still need PhantomJS for these two scripts:

https://github.com/code-dot-org/code-dot-org/blob/f1b92f1190299e4244c374b8535fec552db0c75d/apps/script/generateLevelImages.js
https://github.com/code-dot-org/code-dot-org/blob/d344587164a1671898b827852c72e0e892918d79/lib/cdo/pegasus/screencap.js

(It's not clear that the second one is used.)  I'll go ahead and remove `karma-phantomjs-launcher`!

**Comment by joshlory** [comment]
I need to fix up a number of tests that pass in PhantomJS but fail when run with `BROWSER=Chrome`.
For example:
```
BROWSER=Chrome grunt karma:entry --entry ./test/integration/levelTests.js --grep "receives movementX and movementY in mousemove events"
```

**Comment by joshlory** [comment]
Still working on this one, sorry for the delay!

**Pull request closed without merging**